src/storage/src/hummock/compactor/compactor_runner.rs

```diff
@@ -369,13 +370,25 @@ pub async fn compact(
     let all_ssts_are_blocked_filter = sstable_infos
         .iter()
         .all(|table_info| table_info.bloom_filter_kind() == BloomFilterType::Blocked);
+
+    let delete_ratio = sstable_infos
+        .iter()
+        .map(|table_info| table_info.stale_key_count)
+        .sum::<u64>()
+        * 100
+        / sstable_infos
+            .iter()
+            .map(|table_info| table_info.total_key_count)
```

Comment:
Can `total_key_count` be zero? Better use `stale_key_count * 100 >= total_key_count * FAST_COMPACT_MAX_DELETE_RATIO` for the check.

Reply:
Emm, it can not be. 
But I think we shall check to avoid other bug cause these case panic

Reply:
I vaguely remember that if the SST only contains range tombstone, the `total_key_count` can be 0.